packages/web/src/pages/oauth-login-page/OAuthLoginPage.module.css

```diff
@@ -1,12 +1,16 @@
-.bgWhite {
-  background-color: var(--harmony-static-white);
-}
-
 .wrapper {
-  margin: 0 auto;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
   justify-content: center;
-  width: 375px;
   display: flex;
+  overflow: scroll;
+}
+
+.popup {
```

Comment:
Maybe use `--harmony-bg-white`, so that it's not stuck on white if the user is in dark mode?
Only matters if the rest of this supports themes though.

Reply:
og design had it static white, so i kept as is!